charmcraft.yaml

```diff
@@ -19,6 +19,8 @@ parts:
       - -requirements.txt
     prime:
       - snap_revisions.json
+      - scripts
+      - templates
```

Comment:
:thinking: wasn't templates there already!

Reply:
Wow... how did the logrotate work without logrotate templates in prime? o_0